controller/array_action/array_mediator_xiv.py

```diff
@@ -238,17 +230,21 @@ def _get_vol_by_wwn(self, volume_id):
 
     def delete_volume(self, volume_id):
```

Comment:
Lets change it to delete_object also

Reply:
its a bit harder, not the same calls and verifiers

Reply:
OK.